app/assets/stylesheets/reviews.scss

```diff
@@ -6,6 +8,151 @@
   border-radius: 0;
 }
 
+/* My Reviews CSS */
+
 #review-label {
   padding-bottom: 10px;
-}
\ No newline at end of file
+}
+
+.myreview-container {
+  background: #FFFFFF;
+  box-shadow: 0px 1px 2px 0px #B8B8B8;
+  border-radius: 4px;
+  margin-left:   0px;
+  margin-right:  0px;
+  margin-top:    5px;
+  margin-bottom: 5px;
+  padding: 40px 40px;
+}
+
+#addreview-container {
```

Comment:
We want to avoid using id selectors in CSS. [see here](http://cssguidelin.es/#ids-in-css) for more information and honestly, I'd read that whole page. It highlights how CSS should be more modular and abstract rather than ultra specific. We really ought to refactor a lot of our styles to reflect that philosophy, so we can start on the "My Reviews" page.

Instead of seeing each element as needing its own CSS, look at it more broadly as the different, similar objects that are on the page. Challenge yourself to see how few selectors and lines you use.